cli/cmd/cli_state.go

```diff
@@ -424,6 +424,11 @@ func (c *cliState) envs() []string {
 		fmt.Sprintf("LW_API_KEY=%s", c.KeyID),
 		fmt.Sprintf("LW_API_SECRET=%s", c.Secret),
 		fmt.Sprintf("LW_API_TOKEN=%s", c.Token),
+		fmt.Sprintf("LW_NONINTERACTIVE=%v", c.nonInteractive),
+		fmt.Sprintf("LW_NOCACHE=%v", c.noCache),
+		fmt.Sprintf("LW_NOCOLOR=%s", os.Getenv("NO_COLOR")),
+		fmt.Sprintf("LW_LOG=%s", c.LogLevel),
+		fmt.Sprintf("LW_JSON=%s", c.jsonOutput),
```

Comment:
What about:

```
LW_DEBUG
LW_PROFILE
LW_ORGANIZATION
```

To represent:
```
--debug
--profile
--organization
```

...respectively?

Reply:
`LW_ORGANIZATION` yes!!!

`LW_PROFILE` I am not sure. I think the CLI has the logic to parse the profile and pass the necessary information to the component, the problem I have is for example, if we pass the profile `foo` via `--profile` but also change the sub-account via `--subaccount`, how does the component know when to use the profile and when to use the rest of the flags?

Do you think we should still pass it?

Reply:
@afiune , I think you are spot on here ( common sense over completeness )...